sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/CoreToCodegenBridgeUtils.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+package com.azure.messaging.servicebus.administration.implementation;
+
+import java.time.Duration;
+
+/**
+ * Utility class that handles functionality not yet available in azure-core that generated code requires.
+ */
+public final class CoreToCodegenBridgeUtils {
+
+    /**
+     * Converts a {@link Duration} to a string in ISO-8601 format with support for a day component.
+     *
+     * @param duration The {@link Duration} to convert.
+     * @return The {@link Duration} as a string in ISO-8601 format with support for a day component, or null if the
+     * provided {@link Duration} was null.
+     */
+    public static String durationToStringWithDays(Duration duration) {
```

Comment:
do we have sufficient test coverage for it?
Also, would it make sense to move it to azure-xml or azure-core?